modeltranslation/translator.py

```diff
@@ -157,7 +157,8 @@ def add_translation_fields(model, opts):
     model_empty_values = getattr(opts, 'empty_values', NONE)
     for field_name in opts.local_fields.keys():
         field_empty_value = parse_field(model_empty_values, field_name, NONE)
-        for lang in mt_settings.AVAILABLE_LANGUAGES:
+        available_languages = getattr(opts, 'languages', mt_settings.AVAILABLE_LANGUAGES)
+        for lang in available_languages:
```

Comment:
```suggestion
        for lang in opts.available_languages:
```

And add property `available_languages` to `TranslationOptions`.

Reply:
Updated code.